site/src/pages/RoadmapPage/ImportZot4PlanPopup.tsx

```diff
@@ -63,7 +66,7 @@ const ImportZot4PlanPopup: FC = () => {
     try {
       // Use the backend route to try to obtain the formatted schedule
       await obtainImportedRoadmap(scheduleName, studentYear);
-      // Success; hide the modal
+      saveRoadmap();
```

Comment:
You seem to call the save function, but the global state doesn't seem to be updated yet. Therefore, reloading the page after an import is not showing the updated planner.

screenshot of network request after importing a new "test_migration" schedule from z4p:
<img width="623" alt="image" src="https://github.com/user-attachments/assets/00b0dbf3-38bf-4f28-a6c7-9ae2f00fd31d" />